### PR TITLE
Update binary location

### PR DESCRIPTION
**Pull request by shriya-deshmukh**
# Problem Statement
- Updated component setup script location.

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [x] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: conf/config.yaml.sample [hunk 1]
```diff
@@ -69,7 +69,7 @@ cortx:
       - tcp://hax.seagate.com:28000
   csm:
     auth_admin: admin
-    csmauth_secret: auth-secret
+    csmauth_secret: csmauth-secret
```
Comment (nitin-seagate) [inline conf/config.yaml.sample:72]: This file is old, please use latest file from Tanuja..

**Review by nitin-seagate — APPROVED** [review]
LGTM

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: conf/config.yaml.sample (hunk 1 above)
Comment (shriya-deshmukh) [reply conf/config.yaml.sample]: Update key as per latest config file.

`auth_secret: csmauth-secret`

File: conf/config.yaml.sample [hunk 2]
```diff
@@ -69,7 +69,7 @@ cortx:
       - tcp://hax.seagate.com:28000
   csm:
     auth_admin: admin
-    csmauth_secret: auth-secret
+    auth_secret: csmauth-secret
     mgmt_admin: cortxadmin
     mgmt_secret: mgmt-secret
```
Comment (nitin-seagate) [inline conf/config.yaml.sample:74]: Please change this:
`mgmt_secret: csm-mgmt-secret`

File: conf/config.yaml.sample [hunk 3]
```diff
@@ -69,7 +69,7 @@ cortx:
       - tcp://hax.seagate.com:28000
   csm:
     auth_admin: admin
-    csmauth_secret: auth-secret
+    auth_secret: csmauth-secret
```
Comment (nitin-seagate) [inline conf/config.yaml.sample:72]: Please change this:
`auth_secret: csm-auth-secret`

**Review by nitin-seagate — CHANGES_REQUESTED** [review]

File: conf/config.yaml.sample (hunk 1 above)
Comment (nitin-seagate) [reply conf/config.yaml.sample]: Nope its wrong

Comment (nitin-seagate) [reply conf/config.yaml.sample]: Change S3-Auth secret as follows:
`auth_secret: s3-auth-secret`

**Review by nitin-seagate — APPROVED** [review]
LGTM

File: conf/config.yaml.sample (hunk 1 above)
Comment (shriya-deshmukh) [reply conf/config.yaml.sample]: Updated secrets keys to:
csm-auth-secret
csm-mgmt-secret
s3-auth-secret

File: conf/config.yaml.sample (hunk 3 above)
Comment (shriya-deshmukh) [reply conf/config.yaml.sample]: Done.

**Review by 83bhp — APPROVED** [review]

**Pull request merged**